eslintrc.js

```diff
@@ -0,0 +1,110 @@
+module.exports = {
+  root: true,
+  ignorePatterns: [
+    'dist',
+    'coverage',
+    'node_modules',
+  ],
+  parserOptions: { ecmaVersion: 'latest' },
+  env: { es6: true },
+  plugins: ['@stylistic', 'import-newlines'],
+  overrides: [
+    {
+      files: ['*.ts'],
+      extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/recommended',
+        'plugin:@angular-eslint/recommended',
+        'plugin:@angular-eslint/template/process-inline-templates'
+      ],
+      rules: {

```

Comment:
Ideally these rules should be part of a plugin similar to `@angular-eslint/recommended`. Also, for the style-based rules, we would consider a prettier config so that prettier can be used with this without conflict.

Reply:
Most rules here are overrides to plugin rules, no need to break it out since into another module as thats getting too fine grained.

I'm not considering prettier at all unless we have a valid use case for it beyond quoting and whitespacing as eslint handles it. If you want auto correcting then please set up your IDE appropriately. Our lint rules should be the source the of truth as it is what will be ran within our pipelines and is what is ran today when deploying through Jenkins's .

Reply:
We can discuss using prettier plugins outside of this PR if we wish to leverage prettier in the future beyond basic IDE level configuration.